sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -234,6 +248,11 @@ export enum OfferingDeliveryOptions {
   Blended = "blended",
 }
 
+export enum MaximumFundedWeeksForAviationOfferingCredentials {
+  CommercialPilotTraining = 17,
+  InstructorsRatingAndEndorsements = 13,
+}
+
```

Comment:
This can be removed if the below suggestion is accepeted.
https://github.com/bcgov/SIMS/pull/5115#discussion_r2325842766